mipidsi/src/lib.rs

```diff
@@ -301,15 +301,15 @@ where
     ///
     /// Returns `true` if display is currently set to sleep.
     ///
-    pub fn is_sleeping<D: DelayUs<u32>>(&self) -> bool {
+    pub fn is_sleeping<D: DelayNs>(&self) -> bool {
```

Comment:
Not directly to this PR, but the type parameter should be removed:
```suggestion
    pub fn is_sleeping(&self) -> bool {
```